src/main/java/core/org/akaza/openclinica/dao/hibernate/JobDetailDao.java

```diff
@@ -44,5 +44,11 @@ public JobDetail findByUuid(String uuid) {
         return (JobDetail) q.uniqueResult();
     }
 
-
+    public List<JobDetail> findStudyIdWithStatus(int studyId, Enum status) {
```

Comment:
can you rename this to findByStudyIdAndStatus?

Reply:
yup!